common/scripted_effects/00_decisions_effects.txt

```diff
@@ -206,17 +206,36 @@ convert_whole_realm_to_feudalism_effect = {
 		}
 		change_government = feudal_government
 	}
-	##You're an independent Muslim? Convert to clan!
+	## Warcraft: You're an independent orc? Convert to clan!
 	else_if = {
 		limit = {
 			is_independent_ruler = yes
 
-			# Warcraft
-			always = no
-			# faith = { religion_tag = islam_religion }
+			culture_group = orc_group
+		}
+		change_government = clan_government
+	}
+
+	## Warcraft: You're an independent dwarf? Convert to clan!
+	else_if = {
+		limit = {
+			is_independent_ruler = yes
+
+			culture_group = dwarf_group
+		}
+		change_government = clan_government
+	}
+
+	## Warcraft: You're an independent tauren? Convert to clan!
+	else_if = {
+		limit = {
+			is_independent_ruler = yes
+
+			culture_group = tauren_group
 		}
 		change_government = clan_government
 	}
```

Comment:
You should use scripted triggers, `Warcraft-Guardians-of-Azeroth-2\common\scripted_triggers`. It's a way to create condition presets. Like this entire code could look like...
```
else_if = {
	limit = {
		is_independent_ruler = yes

		uses_clan_government_trigger = yes
	}
	change_government = clan_government
}
```
Where `uses_clan_government_trigger` is a scripted trigger checking are you either an orc, dwarf or tauren.

Reply:
Also, better place this government trigger in a new file, something like `wc_government_triggers.txt`

Reply:
Better resolve this first, I'll review the rest later.